app/src/main/java/edu/nextstep/camp/calculator/CalculationHistoryAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package edu.nextstep.camp.calculator
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import edu.nextstep.camp.calculator.databinding.ItemResultBinding
+import edu.nextstep.camp.domain.CalculationHistory
+
+class CalculationHistoryAdapter: ListAdapter<CalculationHistory, CalculationHistoryAdapter.CalculationHistoryViewHolder>(CalculationHistoryDiffCallback()) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CalculationHistoryViewHolder {
+        return CalculationHistoryViewHolder.create(parent)
+    }
+
+    override fun onBindViewHolder(holder: CalculationHistoryViewHolder, position: Int) {
+        holder.showCalculationHistory(getItem(position))
+    }
+
+    class CalculationHistoryViewHolder(private val binding: ItemResultBinding): RecyclerView.ViewHolder(binding.root) {
```

Comment:
```suggestion
    class CalculationHistoryViewHolder private constructor(private val binding: ItemResultBinding): RecyclerView.ViewHolder(binding.root) {
```
펙토리 함수를 잘 만들어주셨네요 👍 
펙토리 함수를 만드신 의도를 살려서, 외부에서는 생성자로 객체를 만들지 못하도록 변경해보면 어떨까요?